x-pack/plugins/security_solution/public/detections/containers/detection_engine/alerts/types.ts

```diff
@@ -48,6 +48,10 @@ export interface AlertsIndex {
   index_mapping_outdated: boolean;
 }
 
+export interface HostIsolationIndex {
```

Comment:
This name `HostIsolationIndex` and its interface definition is a bit confusing to me. Looking at `AlertIndex` above, it seems as if it should be an interface that allows you to define a Index name?  so should `action` property below be `name`?

Reply:
I think we spoke on this one and you indicated this is really not an index, but rather the response from the isolate API. can you rename it or move it to where the API response types are/will be defined?

Reply:
yup! will rename + move!